recipes/cpprestsdk/all/conanfile.py

```diff
@@ -0,0 +1,180 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class CppRestSDKConan(ConanFile):
+    name = "cpprestsdk"
+    description = "A project for cloud-based client-server communication in native code using a modern asynchronous " \
+                  "C++ API design"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Microsoft/cpprestsdk"
+    topics = ("conan", "cpprestsdk", "rest", "client", "http")
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "exclude_websockets": [True, False],
+        "exclude_compression": [True, False]
```

Comment:
I would rename these options to `websockets` and `compression` and make them `True` by default.
This avoids double negation.

Reply:
The challenge is the cpprestsdk use the exclude in thier cmake [here](https://github.com/conan-io/conan-center-index/pull/1191/files#diff-863846ca4347706fdc27ba1f744206b9R83-R84)

I would have to negate the setting when configuring cmake

Reply:
That's just adding a `not` in front of the option.
I just want to avoid confusion due to the double negation.

Reply:
My intention was to keep it similar to the bincrafters version I started with... however thats certainly not the case anymore.

I corrected the logic